### PR TITLE
minor changes in support of GridAsPoint

### PR DESCRIPTION
**Pull request by haileyajohnson**

### CODE REVIEW

File: cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java [hunk 1]
```diff
@@ -139,6 +142,41 @@ private class CoverageAsFeatureDatasetPoint extends ucar.nc2.ft.point.PointDatas
     }
   }
 
+  private class CoverageAsStationProfileCollection extends StationProfileCollectionImpl {
+    private VarGroup varGroup;
+
+    CoverageAsStationProfileCollection(VarGroup varGroup) {
+      super(varGroup.name + " AsStationFeatureCollection", varGroup.dateUnit, varGroup.zUnit);
```
Comment (tdrwenski) [inline cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java]:
```suggestion
      super(varGroup.name + " AsStationProfileCollection", varGroup.dateUnit, varGroup.zUnit);
```

File: cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java [hunk 2]
```diff
@@ -139,6 +142,41 @@ private class CoverageAsFeatureDatasetPoint extends ucar.nc2.ft.point.PointDatas
...
+    @Override
+    public PointFeatureCCIterator getNestedPointFeatureCollectionIterator() throws IOException {
+      return null;
+    }
+
+    @Override
+    protected StationHelper createStationHelper() throws IOException {
+      StationHelper helper = new StationHelper();
+      String name = String.format("GridPointRequestedAt[%s]", LatLonPoints.toString(latLonPoint, 3));
+      name = StringUtil2.replace(name.trim(), ' ', "_");
```
Comment (tdrwenski) [inline cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java]: does `.trim()` do anything here? I thought it only removed leading/trailing whitespace and it doesn't look like `name` has any

**Review by tdrwenski — APPROVED** [review]

File: cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java (hunk 2 above)
Comment (haileyajohnson) [reply cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java]: I'm not sure if it's possible for `LatLongPoints.toString` to return trailing whitespace...

Comment (tdrwenski) [reply cdm/core/src/main/java/ucar/nc2/ft2/coverage/writer/CoverageAsPoint.java]: won't the string be "GridPointRequestedAt[LatLonStuff]" ending with "]"? either way its a minor thing

**Pull request merged**